### DISCUSSION TITLE
Disco S3 and Enterprise talents

### DISCUSSION THREAD

**Discussion by Felderburg** [bcholmes/StarTrek2d20#158] [General]
2 questions:

1. Someone on the forums or discord noted that there was a talent in the enterprise pack that wasn't in either of our tools, so I bought the pack. Sato does indeed have a unique talent, although it is arguably a canon-character-specific talent. Should it be added here?

2. Disco S3 isn't listed as a source, and it does have some new species and talents. Once it gets added ( which I shoooooould be able to do) should it be added in the S1S2 source item, or on its own? There is an argument for both, given the events in the show that mark a clear split in the series from S1S2 to S3+.

**Comment by bcholmes** [comment]
1. I included Lorca's Menagerie talent (which is non-standard), so it'd be consistent to include Sato's unique talent.
2. Hm. I'd probably go for making S3 separate, 'cause of time jump stuff.
